docs/01-app/02-guides/custom-server.mdx

```diff
@@ -87,16 +87,17 @@ const app = next({})
 
 The above `next` import is a function that receives an object with the following options:
 
-| Option       | Type               | Description                                                                         |
-| ------------ | ------------------ | ----------------------------------------------------------------------------------- |
-| `conf`       | `Object`           | The same object you would use in `next.config.js`. Defaults to `{}`                 |
-| `dev`        | `Boolean`          | (_Optional_) Whether or not to launch Next.js in dev mode. Defaults to `false`      |
-| `dir`        | `String`           | (_Optional_) Location of the Next.js project. Defaults to `'.'`                     |
-| `quiet`      | `Boolean`          | (_Optional_) Hide error messages containing server information. Defaults to `false` |
-| `hostname`   | `String`           | (_Optional_) The hostname the server is running behind                              |
-| `port`       | `Number`           | (_Optional_) The port the server is running behind                                  |
-| `httpServer` | `node:http#Server` | (_Optional_) The HTTP Server that Next.js is running behind                         |
-| `turbo`      | `Boolean`          | (_Optional_) Enable Turbopack                                                       |
+| Option              | Type               | Description                                                                         |
+| ------------------- | ------------------ | ----------------------------------------------------------------------------------- |
+| `conf`              | `Object`           | The same object you would use in `next.config.js`. Defaults to `{}`                 |
+| `dev`               | `Boolean`          | (_Optional_) Whether or not to launch Next.js in dev mode. Defaults to `false`      |
+| `dir`               | `String`           | (_Optional_) Location of the Next.js project. Defaults to `'.'`                     |
+| `quiet`             | `Boolean`          | (_Optional_) Hide error messages containing server information. Defaults to `false` |
+| `hostname`          | `String`           | (_Optional_) The hostname the server is running behind                              |
+| `port`              | `Number`           | (_Optional_) The port the server is running behind                                  |
+| `httpServer`        | `node:http#Server` | (_Optional_) The HTTP Server that Next.js is running behind                         |
+| `turbopack`/`turbo` | `Boolean`          | (_Optional_) Enable Turbopack (enabled by default)                                  |
```

Comment:
I'm fine supporting both fields, but should we just document `turbopack` version and maybe annotate any `turbo` field in the typescript types as deprecated?

Reply:
done, dropped from the docs and deprecated the property in source